osx/create-osx-dmg.sh

```diff
@@ -70,7 +70,7 @@ fi
 # create the initial dmg
 echo "-- Create volume"
 du -sm "$work_dir/${STAGING_DIR}" | awk '{print $1}' > "$work_dir/_size"
-expr "$(cat "$work_dir/_size")" + 99 > "$work_dir/_size"
+expr "$(cat "$work_dir/_size")" + 150 > "$work_dir/_size"
```

Comment:
Can you add some more info?
Why is that needed?

Reply:
getted err size to small